.github/workflows/test-codeql.yml

```
@@ -0,0 +1,64 @@
+      with:
+        version: 8
+
+    - name: Install dependencies
+      run: pnpm install
+
+    - name: Test CodeQL Initialization
+      uses: github/codeql-action/init@v3
+      with:
+        languages: javascript-typescript
+        config-file: ./.github/codeql/codeql-config.yml
+        disable-default-queries: true
```

actionlint (1.7.7)

input "disable-default-queries" is not defined in action "github/codeql-action/init@v3". available inputs are "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)